systemctl_mqtt/__init__.py

```diff
@@ -222,6 +230,17 @@ async def publish_homeassistant_device_config(
                     unit_name=unit_name
                 ),
             }
+        for unit_name in self._controlled_system_unit_names:
+            config["components"]["unit/system/" + unit_name + "/restart"] = {  # type: ignore
+                "unique_id": f"{unique_id_prefix}-unit-system-{unit_name}-restart",
+                "object_id": f"{hostname}_unit_system_{unit_name}_restart",
+                "name": f"{unit_name} restart",
+                "platform": "button",
+                # TODO: Rename get_system_unit_active_state_mqtt_topic to get_system_unit_restart_mqtt_topic ?
+                "state_topic": self.get_system_unit_active_state_mqtt_topic(
```

Comment:
```suggestion
                "command_topic": self.get_system_unit_restart_mqtt_topic(
```

mqtt buttons in home assistant trigger via "command_topic".

Reply:
done, thanks!